packages/team-management/package.json

```diff
@@ -32,14 +32,12 @@
   "author": "OpenSRP Engineering",
   "license": "Apache-2.0",
   "private": false,
-  "dependencies": {
-    "@opensrp/reducer-factory": "^0.0.11",
-    "@opensrp/server-service": "^0.0.11"
-  },
   "devDependencies": {
+    "@opensrp/reducer-factory": "^0.0.11",
```

Comment:
I think that `@opensrp/reducer-factory` is not a peer here, its an actual dependency.